<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vectograph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appresources.qrc">
    <normaloff>:/images/images/Palette.png</normaloff>:/images/images/Palette.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,10">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbtnGraphicalMenuActivation">
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnColorPanelAct">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnAppMenuActivation">
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="grpbxGraphicalMenu">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbtnPointer">
           <property name="toolTip">
            <string>Pointer</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/pointer.png</normaloff>:/images/images/pointer.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnSelectRegion">
           <property name="toolTip">
            <string>Select region</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/selectRegion.png</normaloff>:/images/images/selectRegion.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnPencil">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/pencil.png</normaloff>:/images/images/pencil.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnLine">
           <property name="toolTip">
            <string>Line</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/line.png</normaloff>:/images/images/line.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnRect">
           <property name="toolTip">
            <string>Rectangle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/rectangle.png</normaloff>:/images/images/rectangle.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnCircle">
           <property name="toolTip">
            <string>Circle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/circle.png</normaloff>:/images/images/circle.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnEllipse">
           <property name="toolTip">
            <string>Ellipse</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/ellipse.png</normaloff>:/images/images/ellipse.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnPath">
           <property name="toolTip">
            <string>Path</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/path.png</normaloff>:/images/images/path.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnPolyline">
           <property name="toolTip">
            <string>Polyline</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/polyline.png</normaloff>:/images/images/polyline.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnPolygon">
           <property name="toolTip">
            <string>Polygon</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/star.png</normaloff>:/images/images/star.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnText">
           <property name="toolTip">
            <string>Text</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/oldPen.png</normaloff>:/images/images/oldPen.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="MyGraphicsView" name="gvGraphRegion" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>700</height>
           </size>
          </property>
          <zorder>bottomPanel</zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="bottomPanel">
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="ColorSelect" name="colorSelect" native="true">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>128</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>400</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Object</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>line width</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="spinBoxLineWidth">
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="minimum">
                      <double>1.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="minimumSize">
                      <size>
                       <width>64</width>
                       <height>16</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Opacity</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="sliderOpacity">
                     <property name="layoutDirection">
                      <enum>Qt::LeftToRight</enum>
                     </property>
                     <property name="maximum">
                      <number>100</number>
                     </property>
                     <property name="value">
                      <number>100</number>
                     </property>
                     <property name="tracking">
                      <bool>true</bool>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>fill</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="sliderFillStroke">
                     <property name="maximum">
                      <number>1</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <pointsize>14</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>stroke</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Tab 2</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="deleteBtn">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpbxAppMenu">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbtnNewFile">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create New SVG File for your creative project !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/newFile.png</normaloff>:/images/images/newFile.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnOpenFile">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open your favourite or recent files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/openFile.png</normaloff>:/images/images/openFile.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnSaveFile">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save your file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/SaveFile.png</normaloff>:/images/images/SaveFile.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnAppOptions">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure &amp;quot;Vectograph&amp;quot; by your taste!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnHelp">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;We can help you.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appresources.qrc">
             <normaloff>:/images/images/warning.png</normaloff>:/images/images/warning.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>pbtnNewFile</zorder>
        <zorder>pbtnHelp</zorder>
        <zorder>pbtnOpenFile</zorder>
        <zorder>pbtnSaveFile</zorder>
        <zorder>pbtnAppOptions</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorSelect</class>
   <extends>QWidget</extends>
   <header>colorselect.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QWidget</extends>
   <header>mygraphicsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="appresources.qrc"/>
 </resources>
 <connections/>
</ui>
